includes/Abstracts/DokanBaseRESTController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace WeDevs\Dokan\Abstracts;
+
+use WP_REST_Controller;
+
+/**
+ * Base REST Controller for Dokan
+ *
+ * @since DOKAN_PRO_SINCE
+ *
+ * @package dokan
+ */
+abstract class DokanBaseRESTController extends WP_REST_Controller {
```

Comment:
Pls rename the class to `DokanRESTBaseController` to make semantic with `DokanRESTAdminController`, `DokanRESTVendorController` and `DokanRESTCustomerController`